### DISCUSSION TITLE
Formily 3.0 思考！

### DISCUSSION THREAD

**Discussion by kaoding** [alibaba/formily#2457] [Ideas]
- 现在的 Formily 2.0 结合 designable 在表单引擎和表单设计器上做的相当不错。如果不纵向发展（向后端结合业务，这个可能团队几乎是不考虑），我个人只能认为是在这个基础小修小改（Formily 2.0在架构设计、性能、API合理性等方面做的相当好）。
- 但看到知乎上作者有一些想法：[描绘现实世界的桥梁--Formily](https://zhuanlan.zhihu.com/p/407347096 )，应该把它变成更大的想法。
- 通用 json 来配置所有页面（而不紧紧是表单，现在好想也可以，但定位还是表单），所以 Formily 3.0 应该是可以通用json配置页面的(参考：[amis](https://baidu.gitee.io/amis/zh-CN/docs/index)），或者又要改名（叫：Pageily，呵呵）！

**Comment by janryWang** [comment]
你说的formily3.0应该会是另一个产品，2.0我的想法是持续维护3年以上，目前这样的架构，3年的生命周期应该不会有太大问题，至于另一个产品，目前我还在酝酿，不过暂时也没时间想太多，先把2.0整个生态完善建立起来，站稳脚步，才能一步步向前吧
